benefits/enrollment/views.py

```diff
@@ -125,6 +184,54 @@ def index(request):
         return TemplateResponse(request, TEMPLATE_INDEX, context)
 
 
+def _enroll(client: Client, group_id, funding_source_id, expiry_date=None):
```

Comment:
This helper function feels extraneous, it just calls a single method on the `client` and forwards all the args. I think we should unwind this one and go back to calls on the `client` directly, based on noting a few bugs above in its usage.

Reply:
Done in rebase (dropped the commit)